CMakeLists.txt

```diff
@@ -126,7 +126,8 @@ option(GGML_RVV              "ggml: enable rvv"              ON)
 option(GGML_VXE              "ggml: enable vxe"              ON)
 
 option(GGML_CPU_ALL_VARIANTS "ggml: build all variants of the CPU backend (requires GGML_BACKEND_DL)" OFF)
-set(GGML_CPU_ARM_ARCH "" CACHE STRING "ggml: CPU architecture for ARM")
+set(GGML_CPU_ARM_ARCH        "" CACHE STRING "ggml: CPU architecture for ARM")
+set(GGML_CPU_POWERPC_CPUTYPE "" CACHE STRING "ggml: CPU type for PowerPC")
```

Comment:
Maybe we should consolidate all of these into a single option in the future, but it's not important right now.

Reply:
I'm happy to propose something once I've done more of a deep dive into other architectures. I'm not yet that familiar with the nuances of the other architectures, or how they would map. For example, `-march` vs. PowerPC's  `-mcpu` and `-mtune`.

Reply:
I am not sure either, but probably an option to set the value of `-march` or `-mcpu` would work well enough in most cases.